src/rest/RequestHandler.js

```diff
@@ -269,6 +266,19 @@ class RequestHandler {
           limit = this.limit;
           timeout = this.reset + this.manager.client.options.restTimeOffset - Date.now();
         }
+
+        this.manager.client.emit(
+          DEBUG,
+          `Hit a 429 while executing a request.
+    Global  : ${isGlobal}
+    Method  : ${request.method}
+    Path    : ${request.path}
+    Route   : ${request.route}
+    Limit   : ${limit}
+    Timeout : ${timeout}ms
+    Sublimit: ${sublimitTimeout ? `${sublimitTimeout}ms` : false}`,
```

Comment:
Shouldn't we show `None`/`none` instead of just `false`?
```suggestion
    Sublimit: ${sublimitTimeout ? `${sublimitTimeout}ms` : 'None'}`,
```